test/rpc/helpers.js

```diff
@@ -54,7 +54,7 @@ module.exports = {
     return server
   },
 
-  params (method, params) {
+  params (method, params = []) {
```

Comment:
Ah, that's a good addition, with the default array initialization.

Reply:
Thanks!